examples/vmwithcontracts/actions/deploy.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package actions
+
+import (
+	"context"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/units"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/consts"
+	"github.com/ava-labs/hypersdk/examples/vmwithcontracts/storage"
+	"github.com/ava-labs/hypersdk/keys"
+	"github.com/ava-labs/hypersdk/state"
+	"github.com/ava-labs/hypersdk/x/programs/runtime"
+
+	mconsts "github.com/ava-labs/hypersdk/examples/vmwithcontracts/consts"
+)
+
+var _ chain.Action = (*Deploy)(nil)
+
+type Deploy struct {
+	ProgramID    runtime.ProgramID `json:"programID"`
+	CreationInfo []byte            `json:"creationInfo"`
+	address      codec.Address
+}
+
+func (*Deploy) GetTypeID() uint8 {
+	return mconsts.DeployID
+}
+
+func (t *Deploy) StateKeys(_ codec.Address, _ ids.ID) state.Keys {
+	if t.address == codec.EmptyAddress {
+		t.address = storage.GetAddressForDeploy(0, t.CreationInfo)
+	}
+	stateKey, _ := keys.Encode(storage.AccountProgramKey(t.address), 36)
+	return state.Keys{
+		string(stateKey): state.All,
+	}
+}
+
+func (*Deploy) StateKeysMaxChunks() []uint16 {
+	return []uint16{storage.BalanceChunks}
```

Comment:
Is this correct or leftover from MorpheusVM? If we are placing the contract bytecode at this key, then this should be `len(CreationInfo) / 64` no?

Reply:
@aaronbuchwald, I saw that this comment was resolved without explaining the resolution. Was there an offline resolution?